.github/workflows/e2e-multicluster-ci.yml

```diff
@@ -116,14 +116,25 @@ jobs:
 
           token=$(kubectl get secret -n fleet-default second-token -o go-template='{{index .data "values" | base64decode}}' | yq eval .token -)
           ca=$(kubectl get secret -n cattle-fleet-system fleet-controller-bootstrap-token -o go-template='{{index .data "ca.crt" | base64decode}}')
+          apiServerURL=$(kubectl get node k3d-upstream-server-0 -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
```

Comment:
This is a change that was necessary for me as I, if I remember correctly, have extended the range of default networks available to docker, which results in a different network being used at the start. It seems to work well this way.

Reply:
I wonder what the difficulties on Mac would be with this line, perhaps I can address them as well?

Reply:
Not sure, maybe it works. Act runs everything in containers? So the address is probably reachable.

Reply:
Yes, it runs everything in containers.